pkg/virt-api/webhooks/validating-webhook/admitters/vmpool-admitter.go

```diff
@@ -169,6 +169,18 @@ func ValidateVMPoolSpec(ar *admissionv1.AdmissionReview, field *k8sfield.Path, p
 		}
 	}
 
+	if spec.UpdateStrategy != nil {
+		if spec.UpdateStrategy.Unmanaged != nil && *spec.UpdateStrategy.Unmanaged {
+			if spec.UpdateStrategy.Proactive != nil || spec.UpdateStrategy.Opportunistic != nil {
+				causes = append(causes, metav1.StatusCause{
+					Type:    metav1.CauseTypeFieldValueInvalid,
+					Message: "unmanaged update strategy cannot be used with proactive or opportunistic update strategy",
+					Field:   field.Child("updateStrategy").String(),
+				})
+			}
+		}
+	}
```

Comment:
What happens if `spec.UpdateStrategy.Unmanaged` is nil but both `spec.UpdateStrategy.Proactive` and `spec.UpdateStrategy.Opportunistic` are non-nil? should this case be valid?